_includes/docs/pe/user-guide/integrations/udp.md

```diff
@@ -13,7 +13,7 @@
 UDP Integration allows to stream data from devices which use a UDP protocol to ThingsBoard and converts payloads of these devices into the ThingsBoard format.
 
 
-**Please note** UDP Integration can be started only as [Remote Integration](/docs/{{peDocsPrefix}}user-guide/integrations/remote-integrations). It could be started on the same machine, where TB instance is running, or you can start in on another machine, that has access over the network to the TB instance.
+**Please note** UDP Integration can be started only as [Remote Integration](/docs/user-guide/integrations/remote-integrations). It could be started on the same machine, where TB instance is running, or you can start in on another machine, that has access over the network to the TB instance.
```

Comment:
You have an obvious error in the first line of the PR. Please START REVIEWING YOUR CODE BEFORE SUBMITTING PRs. 

Why {{peDocsPrefix}} was removed?